packages/app-store/jelly/api/callback.ts

```diff
@@ -0,0 +1,72 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+
+import prisma from "@calcom/prisma";
+
+import getAppKeysFromSlug from "../../_utils/getAppKeysFromSlug";
+import getInstalledAppPath from "../../_utils/getInstalledAppPath";
+import createOAuthAppCredential from "../../_utils/oauth/createOAuthAppCredential";
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  const { code } = req.query;
+  const { client_id, client_secret } = await getAppKeysFromSlug("jelly");
+
+  const result = await fetch(`https://jellyjelly.com/login/oauth/access_token`, {
+    method: "POST",
+    body: JSON.stringify({ code, client_id, client_secret }),
+  });
+
+  if (result.status !== 200) {
+    let errorMessage = "Something is wrong with the Jelly API";
+    try {
+      const responseBody = await result.json();
+      errorMessage = responseBody.error;
+    } catch (e) {}
+
+    res.status(400).json({ message: errorMessage });
+    return;
+  }
+
+  const responseBody = await result.json();
+  console.log(responseBody);
```

Comment:
The response I'm getting back doesn't contain the expiration date only the `access_token`, `scope`, and `token_type`.

Reply:
I removed the expiration since we're not using it right now.